classes/class3/hw3/routes/cat.js

```diff
@@ -0,0 +1,85 @@
+var mongoose = require('mongoose');
+
+var catName = ['Gizmo', 'Beauregard', 'Baby Girl', 'Han Solo', 'Trisha', 'Quat', 'Arbitration', 'Darth', 'Saint Ashlar', 'Willa Catter',
+	'Curtains', 'Seurat', 'Fox', 'Possum' ,'Akeisha','Manuel','Nika','Rossini','Teddy Bear','Katman','Pooker','Jack Sparrow','Gecko','Tanier','Berkeley'];
+var catColors = ['brown', 'red', 'black', 'white', 'grey', 'orange', 'yellow'];
+
+var Cat = mongoose.model('Cat',{
```

Comment:
In the future, you can break this model logic out into its own module. It will help when we start having multiple models each with many more attributes.